styles.css

```diff
@@ -25,6 +25,25 @@ body {
   position: fixed;
 }
 
+.starred-btn-container {
```

Comment:
some fun frameworks for css and styling include tailwind. idk if you've heard of them :) this looks good so far tho